WikiWatershed/model-my-watershed
# Compare View: Fix 100% Forest Cover Scenario

## Overview

Even in urban areas, the compare view Current Condition values were the same as the 100% Forest Cover scenario's. We were unconditionally using the `modified` results, but the 100% Forest Cover results exist under `pc_unmodified`. This PR has the charts use results from the key that corresponds with each scenario. This PR also adds a few tests to confirm the compare charts are using the proper values from the results.

Adding the tests involved updating the mock scenario in `modeling/mocks` which was missing the full result object (it didn't have `pc_unmodified`).

Connects #2322 

### Demo

![screen shot 2017-10-09 at 2 43 44 pm](https://user-images.githubusercontent.com/7633670/31353700-8c4cc812-ad01-11e7-82b4-5345c8ed8bd8.png)

## Testing Instructions

 * Pull this branch and `bundle`
 * Add a scenario to the TR-55 model with a forest modification covering the extent of the AOI
 * Confirm it is similar to the 100% forest scenario results in the compare view for both Runoff and Water Quality
